IRC.md

```diff
@@ -335,6 +336,7 @@ Changes only contain the relevant tag. Setting slow mode to 10 seconds for examp
 - `r9k` is R9K mode. Messages with more than 9 characters must be unique. `0` means disabled, `1` enabled.
 - `subs-only` is subscribers only mode. Only subscribers and moderators can chat. `0` disabled, `1` enabled.
 - `slow` determines how many seconds chatters without moderator privileges must wait between sending messages.
+- `followers-only` is follower only mode. Only followers and moderators can chat. `-1` disabled, `0` enabled.
```

Comment:
This is not incorrect. If you set a duration this holds a positive number.

On a related note, I'm not sure this repostiory is still monitored by twitch staff, as the official documentation is now on dev.twitch.tv and it has been mentioned, that it is no longer related to this repository.